semanticdb/metacp/src/main/scala/scala/meta/internal/metacp/package.scala

```diff
@@ -1,18 +1,37 @@
 package scala.meta.internal
 
-import java.nio.file._
+import java.nio.file.Files
 import scala.meta.io.AbsolutePath
 import scala.tools.asm._
 import scala.tools.asm.ClassReader._
 import scala.tools.asm.tree._
 
 package object metacp {
+  private val SKIP_ALL = SKIP_CODE | SKIP_DEBUG | SKIP_FRAMES
+
+  implicit class XtensionAsmBytes(entry: ReadableClasspathEntry) {
+    def hasScalaSig: Boolean = {
+      val visitor = new HasScalaSigVisitor
```

Comment:
Also, this logic is inconsistent with internal/metacp/Main.scala, which checks for both `ScalaSig` and `Scala` classfile attrs.

Reply:
I've synchronised the two